inc/widgets/author-bio.php

```diff
@@ -25,28 +25,47 @@ function __construct() {
 	function widget( $args, $instance ) {
 
 		global $post;
-		$author_id = $post->post_author;
 
-		$output = '';
+		$authors = array();
+		$bios = '';
 
-		echo $args['before_widget'];
+		if( get_post_meta( $post->ID, 'largo_byline_text' ) )
```

Comment:
I don't think `largo_byline_text()` has been defined in 1.0 yet, so we'll either need to define it or remove this